blog/2023-01-18-introducing-netdata-paid-plans.md

```diff
@@ -0,0 +1,95 @@
+---
+slug: introducing-netdata-paid-plans
+title: Introducing Netdata Paid Plans
+description: All Netdata functionality is and will be available for free in the Free Community Plan. Paid tiers include features targeted for managers, and administrators.
```

Comment:
All Netdata functionality is and will be available for free forever in the Community Plan. Paid tiers include features targeted for businesses and users who would need to customise their monitoring solution with different levels of user access, other notification mechanisms etc.